ui/raidboss/data/triggers/seiryu-ex.js

```diff
@@ -133,20 +133,66 @@
         en: 'Get Tower',
       },
     },
+    {
+      id: 'SeiryuEx Handprint East Move',
+      regex: / 15:\y{ObjectId}:Yama-no-shiki:37E5:Handprint:/,
+      regexDe: / 15:\y{ObjectId}:Yama no Shiki:37E5:Handabdruck:/,
+      condition: function(data) {
+        return data.firstHandprint !== undefined;
+      },
+      infoText: {
+        en: 'Move, East',
```

Comment:
Sorry to be subjectively picky here, but I don't know that the "move" part here really adds much for me.  I think just saying "east" or "west" should be enough.

Also, if this is messing up tts, then I'd suggest adding a specific `tts: { en: 'East', etc }` section here that skips the `<=`.

Reply:
I was originally planning to add a tts section to the original, but could not figure out if I was doing that correctly or not as I had not used tts yet.

I am ok with not adding the move text, it was something I just wanted to test out with and this fight's east/west is clearer and less deadly than o10s' starboard/larboard. In my original version (prior to the master update) of this I was also calling out left/right instead of cardinals as well, but did agree that cardinals should be used as the Yama-no-shiki is always from the north.